cmake/Deps_Unix.cmake

```diff
@@ -89,11 +96,6 @@ if (WITH_ZIM)
         # icu4c as transitive dependency of libzim may not be automatically copied into app bundle
         # so we manually discover the icu4c from homebrew, then find the relevent dylibs
         pkg_check_modules(BREW_ICU REQUIRED IMPORTED_TARGET icu-i18n icu-uc)
```

Comment:
icu-data?

Reply:
Changed to use the path relative to icu4c's prefix like `${icu4c prefix}/lib/libicudata.dylib`, so that there is no need to go through pkgconf.